db/models/comments.model.js

```diff
@@ -0,0 +1,27 @@
+const db = require("../connection");
+const { isValidId } = require('../../utils');
+
+exports.selectCommentsByArticleId = (article_id) => {
+    
+    if (!isValidId(article_id)) {
+      return Promise.reject({ status: 400, msg: "Invalid article ID" })
+    }
+    
+    const sqlQuery = `
+    SELECT 
+        comments.comment_id,
+        comments.votes, 
+        comments.created_at, 
+        comments.author, 
+        comments.body, 
+        comments.article_id  
+    FROM comments
+    LEFT JOIN articles ON comments.article_id = articles.article_id
```

Comment:
consider whether the join here is adding any value to your query! 👀 We just want all the comments where the article id is correct